src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,49 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact.
+ */
+public class Contact {
+
+    private final String validationRegex;
+    
+    public final String value;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.